WindowsServerDocs/remote/remote-access/vpn/ad-ca-vpn-connectivity-windows10.md

```diff
@@ -84,7 +84,7 @@ In this step, you can create OMA-DM based VPNv2 profiles using Intune to deploy
 
 - [Configure VPNv2 Profiles](https://docs.microsoft.com/windows/access-protection/vpn/vpn-conditional-access): The VPN client is now able to integrate with the cloud-based Conditional Access Platform to provide a device compliance option for remote clients. In this step, you configure the VPNv2 profiles with **\<DeviceCompliance> \<Enabled>true\</Enabled>**.
 
-- [Enhancing remote access in Windows 10 with an automatic VPN profile](https://www.microsoft.com/itshowcase/Article/Content/894/Enhancing-remote-access-in-Windows-10-with-an-automatic-VPN-profile): Learn how Microsoft implements Conditional Access for VPN connectivity. VPN profiles contain all the information a device requires to connect to the corporate network, including the authentication methods that are supported and the VPN server that the device should connect to. Changes in Windows 10 Anniversary Update, including Conditional Access and single sign-on, made it possible for us to create our Always-On VPN connection profile. We created the connection profile for domain-joined and Microsoft Intune–managed devices using System Center Configuration Manager console.
+- [Enhancing remote access in Windows 10 with an automatic VPN profile](https://www.microsoft.com/itshowcase/Article/Content/894/Enhancing-remote-access-in-Windows-10-with-an-automatic-VPN-profile): Learn how Microsoft implements Conditional Access for VPN connectivity. VPN profiles contain all the information a device requires to connect to the corporate network, including the authentication methods that are supported and the VPN server that the device should connect to. Changes in Windows 10 Anniversary Update, including Conditional Access and single sign-on, made it possible for us to create our Always-On VPN connection profile. We created the connection profile for domain-joined and Microsoft Intune–managed devices using Microsoft Endpoint Configuration Manager console.
```

Comment:
@shortpatti to help here. This bullet is referencing the MSIT implementation from almost three years ago. While some of that content is probably still applicable, the last line is talking about a hybrid configuration (creating the connection profiles for Intune-managed devices from the ConfigMgr console) - the hybrid service was turned off 1 September 2019. (I'm actively removing hybrid content from the ConfigMgr docs.) So maybe the easy solution here is to just remove the bit about ConfigMgr here (remove the entire last sentence?). The IT Showcase content still talks about it, but it's obviously dated.

Reply:
@shortpatti Do you have any suggestions regarding Aaron's comment? I'm happy to make the changes in the article. Thanks!

Reply:
@aczechowski @shortpatti  I made the changes suggested by Aaron so it would be easier to visualize. Let me know if that is acceptable or how else it should be changed. Thanks.